osu.Game.Rulesets.Catch/Objects/CatchHitObject.cs

```diff
@@ -27,7 +27,9 @@ public abstract class CatchHitObject : HitObject, IHasXPosition, IHasComboInform
         public int ComboIndex { get; set; }
 
         /// <summary>
-        /// The distance for a fruit to to next hyper if it's not a hyper.
+        /// The difference between the distance of the next object 
```

Comment:
"Difference between the distance to the next object and the distance that would have triggered a hyper dash. A value close to 0 indicates a difficult jump (for difficulty calculation)."